### PR TITLE
Custom scheduled report subject

### PR DESCRIPTION
**Pull request by nickpell**
screen shot 2015-08-31 at 3 26 01 pm

@dimagi/product @biyeun cc: @dannyroberts

### CODE REVIEW

File: corehq/apps/reports/models.py [hunk 1]
```diff
@@ -713,7 +716,11 @@ def _get_and_send_report(self, language, emails):
         from corehq.apps.reports.views import get_scheduled_report_response
 
         with localize(language):
-            title = _("Scheduled report from CommCare HQ")
+            title = (
+                _(DEFAULT_REPORT_NOTIF_SUBJECT)
+                if self.email_subject == DEFAULT_REPORT_NOTIF_SUBJECT
+                else self.email_subject
+            )
```
Comment (czue) [inline corehq/apps/reports/models.py]: does the default subject get properly migrated on reads of old models that don't have it?

Comment (nickpell) [inline corehq/apps/reports/models.py]: yes - the ```default``` keyword in ```StringProperty``` ensures that old models will have it

Comment (czue) [inline corehq/apps/reports/models.py]: :+1:

**Comment by czue** [comment]
good from my side. will wait for a :+1: from @dimagi/product

**Comment by nickpell** [comment]
@dimagi/product bump

**Comment by amsagoff** [comment]
good by me

**Pull request merged**